src/repl/scala/tools/nsc/interpreter/ReplGlobal.scala

```diff
@@ -42,5 +42,13 @@ trait ReplGlobal extends Global {
     }
   }
 
+  trait ReplAnalyzer extends Analyzer {
+    override def rootImports(unit: global.CompilationUnit) = super.rootImports(unit)
```

Comment:
I don't get it. What's this for?

Reply:
Widens access so I can actually call it. Maybe I should have just removed `protected`. I had a cake error in the param at first brush. That may have been my last cake error before jumping to scala 3.

Reply:
> Maybe I should have just removed `protected`

👍

Reply:
In my PR to model "import repl history" as root imports, I need this hook anyway.